cmd/record.go

```diff
@@ -308,7 +308,7 @@ func (r *Record) GetCmd() *cobra.Command {
 
 	recordCmd.Flags().StringP("command", "c", "", "Command to start the user application")
 
-	recordCmd.Flags().DurationP("recordTimer", "t", 0, "Timer to stop keploy recorder after a specified time")
+	recordCmd.Flags().Duration("recordTimer", 0, "Timer to stop keploy recorder after a specified time")
```

Comment:
Why have you removed the code for reading this flag?

Reply:
eyy! so i removed the shorthand entirely and kept it as --recordTimer , keeping the shorthand as -r would lead to ambiguity, So please chec